opendevin/runtime/utils/runtime_build.py

```diff
@@ -12,6 +12,10 @@
 import opendevin
 from opendevin.core.logger import opendevin_logger as logger
 
+RUNTIME_IMAGE_REPO = os.getenv(
+    'OD_RUNTIME_RUNTIME_IMAGE_REPO', 'ghcr.io/opendevin/od_runtime'
```

Comment:
isn't 1 RUNTIME enough? 😁

Reply:
Yeah, I'd ideally want to keep all the flexibility here (e.g., We might use a different image registry for the hosted version :D)